### DISCUSSION TITLE
How do I find the repository to which my domain is linked?

### DISCUSSION THREAD

**Discussion by djfm** [community/community#76969] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I have a root GH-pages site on one of my 200+ repos, but I don't remember which one it is anymore.

How can I find out which repository is tied to my domain?

**Comment by milad176** [comment]
Finding the repository linked to a domain can depend on how the domain is configured and what kind of hosting or version control system is in use. Here are a few common scenarios:

GitHub Pages:

If your domain is linked to a GitHub Pages site, you can find the repository by following these steps:

1. Go to the GitHub Pages site associated with your domain (usually in the format `username.github.io` or `organization.github.io`).
   
2. Navigate to the repository by clicking on the repository name, usually located near the top of the GitHub Pages site.

**Comment by djfm** [comment]
Thank you,

I might as well give the domain name, it's gonna be easier showing than
explaining.

It's https://fmdj.fr, and the site can be accessed through
https://djfm.github.io too,
but this immediately redirects to https://fmdj.fr and I'm still as clueless
as to which repository is being used to serve that website...

**Reply by peterroth** [reply, marked as answer]
The fmdj.fr page is hosted from your [djfm.github.io](https://github.com/djfm/djfm.github.io) repository, because the [CNAME file](https://github.com/djfm/djfm.github.io/blob/main/CNAME) in this repository points to fmdj.fr, what is the correct way to configure your custom domain name for Pages, as described in the docs: [Configuring a subdomain](https://docs.github.com/en/pages/configuring-a-custom-domain-for-your-github-pages-site/managing-a-custom-domain-for-your-github-pages-site#configuring-a-subdomain), point 4.

**Comment by djfm** [comment]
Thaaaaank you so much, how did I not think of that! :) awesome!

**Reply by djfm** [reply]
Oh yes of course, sorry I was replying via the e-mail notification and I didn't see that option. Done.

But how did you find it was that repo exactly if you don't mind? is it (or was it) just a convention based on user or org name? I had totally forgotten how I had initially set this up.

I see one of my DNS records points to djfm.github.io but that is not necessarily the name of the repo that has the domain linked to it is it? Doesn't matter much, just trying to understand a bit better.

**Reply by peterroth** [reply]
The [Pages docs](https://pages.github.com/) describe you can create your page in a repo called your-username.github.io, so I just checked what .github.io repos exist and found this one. Then I read the CNAME file and found that it points to the hostname what you described.

**Comment by djfm** [comment]
You are a better reader than I am then, thank you.

**Comment by Shohjaxon-098** [comment]
Response #1

**Comment by Shohjaxon-098** [comment]
Response #2

**Comment by Shohjaxon-098** [comment]
The fmdj.fr page is hosted from your [djfm.github.io](https://github.com/djfm/djfm.github.io) repository, because the [CNAME file](https://github.com/djfm/djfm.github.io/blob/main/CNAME) in this repository points to fmdj.fr, what is the correct way to configure your custom domain name for Pages, as described in the docs: [Configuring a subdomain](https://docs.github.com/en/pages/configuring-a-custom-domain-for-your-github-pages-site/managing-a-custom-domain-for-your-github-pages-site#configuring-a-subdomain), point 4.

**Comment by yasmeenlakhani2020-cmd** [comment]
Check your DNS records to see which GitHub Pages domain they point to, then open the related repository settings under Pages to confirm the custom domain. If your domain is managed through a registrar like [register.domains](https://register.domains) you can review its DNS settings there to identify the connected repository.
